src/lib/containers/entity_finder/details/view/DetailsViewRow.tsx

```diff
@@ -81,14 +79,6 @@ export const DetailsViewRow: React.FunctionComponent<DetailsViewRowProps> = ({
     },
   )
 
-  const { data: modifiedByUserProfile } = useGetUserProfileWithProfilePic(
-    bundle?.entity?.modifiedBy ?? '273950',
-    {
-      enabled: !!bundle,
-      staleTime: 60 * 1000, // 60 seconds
-    },
-  )
```

Comment:
This was just passed into a UserCard, but the UserCard does this on its own now.